### PR TITLE
unfccc items

### PR DESCRIPTION
**Pull request by PatFawbertMills**
Adding 2 places that link through to the UNFCCC survey

### CODE REVIEW

File: src/components/blocks/SearchResultList.tsx [hunk 1]
```diff
@@ -1,13 +1,19 @@
 import { ExternalLink } from "@components/ExternalLink";
 import SearchResult from "./SearchResult";
 import { TMatchedFamily } from "@types";
+import { DOCUMENT_CATEGORIES } from "@constants/documentCategories";
+import Button from "@components/buttons/Button";
 
 type TProps = {
-  category?: string;
+  category?: (typeof DOCUMENT_CATEGORIES)[number];
```
Comment (diversemix) [inline src/components/blocks/SearchResultList.tsx:8]: :grin: I would like this to be its own type ... say `CategoryList` ?

File: src/sites/cclw/components/Articles.tsx [hunk 2]
```diff
@@ -14,6 +14,13 @@ type TArticle = {
 };
 
 const ARTICLES: TArticle[] = [
+  {
+    url: "https://docs.google.com/forms/d/e/1FAIpQLSfpo03ytni3SmEqXfFRW38qz1OGyNgN5HhQF-6fW-QcSLZ67A/viewform",
```
Comment (diversemix) [inline src/sites/cclw/components/Articles.tsx:18]: I'd say we need a single reference to this form in the code - currently we have the URL it two places

**Review by diversemix — APPROVED** [review]
Two comments - first "kind to the next developer" comment - second would be good to do methinks - but your call :smile:

File: src/components/blocks/SearchResultList.tsx (hunk 1 above)
Comment (PatFawbertMills) [reply src/components/blocks/SearchResultList.tsx:9]: I will circle back to this later

**Pull request merged**